### ISSUE TITLE
Deprecate `beer-song` in favor of `bottle-song`

### ISSUE THREAD

**Issue by BNAndras** [exercism/common-lisp#768]
See https://github.com/exercism/problem-specifications/blob/main/exercises/beer-song/.deprecated. bottle-song will need to be ported before we deprecate beer-song. I can work on that by the end of this month if there are no takers before then.

**Comment by verdammelt** [comment]
Feel free. I have not yet looked at 'bottle song' is there any differences other than `s/beer/bottle`?

**Comment by BNAndras** [comment]
Not sure. The plan is to port the exercise afterwards to Haxe, Pyret, Racket, and Vim script. CFML probably needs it too, but I haven't checked.

**Comment by BNAndras** [comment]
Yeah, the main change was beer to bottle. Other tweaks are that the bottles start at 10 instead of 99 and the counts are spelled out like "two" instead of 2. I'll PR `bottle-song` this week.
